plaso/engine/worker.py

```diff
@@ -81,15 +81,11 @@ def __init__(self, parser_filter_expression=None):
       parser_filter_expression (Optional[str]): parser filter expression,
           where None represents all parsers and plugins.
 
-          The parser filter expression is a comma separated value string that
-          denotes a list of parser names to include and/or exclude. Each entry
-          can have the value of:
-
-          * An exact match of a list of parsers, or a preset (see
-            data/presets.yaml for the list of predefined presets).
-          * A name of a single parser (case insensitive), such as msiecf.
-          * A glob name for a single parser, such as '*msie*' (case
-            insensitive).
+          A parser filter expression is a comma separated value string that
+          denotes which parsers should be used. See filters/parser_filter.py
+          for details of the expression syntax.
+
+          Note that preset names in this expression will not be expanded.
```

Comment:
`Note that preset names in this expression will not be expanded.`

Though correct this note does not cover the core of the issue. Change to something in line of:

This function requires a parser filter expression in which preset names have been expanded.